sql/mysqld.cc

```diff
@@ -4781,6 +4781,7 @@ int mysqld_main(int argc, char **argv)
   sys_var_init();
   ulong requested_open_files;
   adjust_related_options(&requested_open_files);
+  my_init_signals();
```

Comment:
Please add a short comment "moved signal initialization here so that PFS thread inherits signal mask or similar"